### DISCUSSION TITLE
Main repository moved from github.com/cyrille-leclerc to github.com/elastic

### DISCUSSION THREAD

**Discussion by cyrille-leclerc** [elastic/opentelemetry-maven-extension#6] [Announcements]
FYI the main repository of the opentelemetry-maven-extension moved from https://github.com/cyrille-leclerc/opentelemetry-maven-extension to https://github.com/elastic/opentelemetry-maven-extension 
cc @cybernetics @oleg-nenashev

**Comment by oleg-nenashev** [comment]
Ack, thanks! Sounds about right
